tests/cpp/test_host_irs.cpp

```diff
@@ -888,6 +888,7 @@ TEST_F(LinearHostIrTest, HostIr) {
 }
 
 TEST_F(LinearHostIrTest, HostIrLinearOut) {
+  GTEST_SKIP();
```

Comment:
Temporarily disabled as it prevents the binary tests running on H100. 

Related: #3996